account_credit_control/security/ir.model.access.csv

```diff
@@ -18,3 +18,6 @@ account_credit_control.ir_model_access_289,credit_control_info_level,account_cre
 account_credit_control.ir_model_access_290,credit_control_fin_user_line,account_credit_control.model_credit_control_line,account.group_account_user,1,0,0,0
 account_credit_control.ir_model_access_291,credit_control_fin_invoice_line,account_credit_control.model_credit_control_line,account.group_account_invoice,1,0,0,0
 account_credit_control.ir_model_access_292,credit_control_fin_manager_line,account_credit_control.model_credit_control_line,account.group_account_manager,1,1,1,1
+account_credit_control.ir_model_access_293,credit_control_fin_user_policy,account_credit_control.model_credit_control_policy,account.group_account_user,1,0,0,0
+account_credit_control.ir_model_access_294,credit_control_fin_invoice_policy,account_credit_control.model_credit_control_policy,account.group_account_invoice,1,0,0,0
+account_credit_control.ir_model_access_295,credit_control_fin_manager_policy,account_credit_control.model_credit_control_policy,account.group_account_manager,1,1,1,1
```

Comment:
The idea here is that you have the permissions for credit control policy isolated from accounting permissions, so I don't see this permission correct. You should instead followed a similar strategy that the one of the view: limiting views to those that have the correct permission.

Reply:
Yes, I think so. This lines added are not necessary.

I'll remove tomorrow.